index.html

```diff
@@ -12,12 +12,12 @@
     }
 
     a:visited {
-      color: #64045c;
+      color: rgb(105, 185, 13);
 
     }
 
     a:hover {
-      color: #CE242B;
+      color: rgb(206, 36, 43);
```

Comment:
This color is not contrast enough, try somthing lighter.

Reply:
ok